modules/mbedtls/crypto_mbedtls.cpp

```diff
@@ -449,3 +436,39 @@ Vector<uint8_t> CryptoMbedTLS::decrypt(Ref<CryptoKey> p_key, Vector<uint8_t> p_c
 	copymem(out.ptrw(), buf, size);
 	return out;
 }
+
+String X509CertificateMbedTLS::save_to_string() const {
+	const mbedtls_x509_crt *crt = &cert;
+	PackedByteArray bytes_out;
+	while (crt) {
+		unsigned char w[4096];
+		size_t wrote = 0;
+		int ret = mbedtls_pem_write_buffer(PEM_BEGIN_CRT, PEM_END_CRT, cert.raw.p, cert.raw.len, w, sizeof(w), &wrote);
+		if (ret != 0 || wrote == 0) {
+			ERR_FAIL_V_MSG(String(), "Error saving certificate '" + itos(ret) + "'.");
+		}
+		PackedByteArray pem_buffer;
+		int len = wrote - 1;
+		pem_buffer.resize(len);
+		copymem(pem_buffer.ptrw(), w, len);
```

Comment:
Can we avoid the triple copy here?

Reply:
Will resolve in next free time ~ 12 hours or another time.

Reply:
I removed one layer of copying.